src/test/kotlin/com/terraformation/backend/accelerator/db/ApplicationStoreTest.kt

```diff
@@ -315,6 +323,418 @@ class ApplicationStoreTest : DatabaseTest(), RunsAsUser {
         assertThrows<AccessDeniedException> { store.fetchAll() }
       }
     }
+
+    @Nested
+    inner class FetchModulesByApplicationId {
+      @Test
+      fun `returns modules with statuses associated with application`() {
+        val prescreenModule =
+            insertModule(
+                name = "Pre-screen",
+                overview = "Pre-screen Overview",
+                phase = CohortPhase.PreScreen,
+            )
+        val applicationModule =
+            insertModule(
+                name = "Application",
+                overview = "Application Overview",
+                phase = CohortPhase.Application)
+        insertModule(name = "Hidden module", phase = CohortPhase.Phase1FeasibilityStudy)
+
+        insertApplicationModule(
+            inserted.applicationId, prescreenModule, ApplicationModuleStatus.Complete)
```

Comment:
With the position column now existing, this probably ought to insert two modules in the same phase with the positions in the opposite order (e.g., insert position 5 first then position 4) to confirm the positions are being honored.